src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -312,9 +312,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly Property IInvocationExpression_IsVirtual As Boolean Implements IInvocationExpression.IsVirtual
```

Comment:
The code is the same as in `IInvocationExpression_IsVirtual` above, consider create a helper.